### ISSUE TITLE
Transaction list?

### ISSUE THREAD

**Issue by Bitminereu** [lanaiscool/ForkNote-webwallet#1]
We have balance and withdrawn but is possible to add transaction list in and out?

thanks

**Comment by streamcoindev** [comment]
yes that is possible however i need to work on that i know someone who has it done but i havent had the time yet to add that feature it will be available but not yet
